pages/preferences-form.js

```diff
@@ -3,31 +3,71 @@ import { useRouter } from "next/router";
 export default function PreferencesForm({
   selectedLocations,
   setSelectedLocations,
+  selectedWeathers,
+  setSelectedWeathers,
+  selectedCompanions,
+  setSelectedCompanions,
 }) {
   const router = useRouter();
-  const location = [
+  const locations = [
     { text: "I'm living my best life on a boat", value: "boat" },
     { text: "I'm in a forest", value: "forest" },
     { text: "I'm in a city", value: "city" },
     { text: "I'm in the mountains", value: "mountains" },
   ];
 
+  const weathers = [
+    { text: "sunny, warm, not too bad", value: "sunny" },
+    { text: "rainy", value: "rainy" },
+    { text: "wuthering heights", value: "windy" },
+    { text: "cloudy", value: "cloudy" },
+  ];
+
+  const companions = [
+    { text: "just me", value: "alone" },
+    { text: "1 person + me", value: "me + 1" },
+    { text: "dog", value: "dog" },
+    { text: "more people", value: "more people" },
+  ];
+
   function submitPreferences() {
     router.push("/");
   }
 
   return (
     <>
-      <h3>I am bored, and...</h3>
-      {location.map((location, i) => (
-        <PreferencesButton
-          key={location.value}
-          preference={location}
-          selectedPreferences={selectedLocations}
-          setPreferences={setSelectedLocations}
-        />
-      ))}
-      <button onClick={submitPreferences}>Show matching games!</button>
+      <>
+        <h3>I am bored, and...</h3>
```

Comment:
you should use a p or span tag here, since it is not really a headline for some text. You can style it later on as you like of course.

Reply:
Thank you for checking my code, Felix :)